packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -6,6 +6,22 @@ title: Okta Identity Engine API Products release notes 2022
 
 ## April
 
+### Weekly release 2022.04.1
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.04.1](#bugs-fixed-in-2022-04-1) | April 6, 2022 |
+
+#### Bugs fixed in 2022.04.1
+
+* An HTTP 500 internal server error sometimes occurred after saving an app instance. (OKTA-483001)
+
+* Active Directory email activation templates that were translated for the Japanese, Korean, and Chinese languages didn’t render correctly. (OKTA-469764)
+
+* An OIDC app couldn’t be updated for an IdP-initiated login with Okta flow when the **Initiate login URI** field in the Admin Console was empty. (OKTA-447112)
+
+* The [Factors Administration API](/docs/reference/api/factor-admin/) property `factorType` couldn't be synchronized in Identity Engine if the equivalent authenticator didn't exist. (OKTA-456613)
```

Comment:
```suggestion
* The Factors Administration API property `factorType` couldn't be synchronized in Identity Engine if the equivalent authenticator didn't exist. (OKTA-456613)
```
Maybe we should remove the link since it's Beta and hidden.

Reply:
Thanks! Pinged Dev mgr and Dev for guidance.

Reply:
Thanks, was determined to remove it from Release Notes.